hack/update-codecgen.sh

```diff
@@ -33,6 +33,14 @@ generated_files=$(
       \) -prune \
     \) -name '*.generated.go')
 
+# Build codecgen binary from Godeps.
+function cleanup {
+  rm -rf "${KUBE_ROOT}/codecgen_binary"
+}
+trap cleanup EXIT
+go build -o codecgen_binary ../../github.com/ugorji/go/codec/codecgen
```

Comment:
I think it would be sufficient to alter GOPATH during the `go build` -- you could say:

```shell
GOPATH=../Godeps/_workspace/src:$GOPATH go build ...
```